radicle-ssh/src/agent/client.rs

```diff
@@ -262,7 +262,7 @@ impl<S: ClientStream> AgentClient<S> {
         // string                  data
         // uint32                  flags
 
-        let mut pk = Vec::new().into();
+        let mut pk: Buffer = Vec::new().into();
```

Comment:
I believe you can do this:
```suggestion
        let mut pk = Buffer::default();
```